test/Test ASUnit.applescript

```diff
@@ -88,7 +88,7 @@ script |ASUnit architecture|
 		script DoesNotInheritFromTopLevel
 			ASUnit's Observer's TEST_FAILED
 		end script
-		shouldRaise(cantGetVar, DoesNotInheritFromTopLevel, �
+		shouldRaise(cantGetVar, DoesNotInheritFromTopLevel, Â
```

Comment:
Can you please revert all these other changes and save the script in Script Editor so that it gets MacRoman encoding? The script as it is in this commit does not compile in Script Editor: if I open it in Script Editor and try to compile I get syntax errors.